backend/src/test/java/com/bikes/backend/controller/BikeControllerTest.java

```diff
@@ -105,9 +140,23 @@ void addBike_returnsABike() throws Exception {
 	@DisplayName("PUT /api/bikes")
 	class testPutBike {
 
+		@Test
+		@DisplayName("...should return 'Unauthorized' (401) if the user is not logged in")
+		void updateBike_returns403IfTheUserIsNotLoggedIn() throws Exception {
+			mockMvc.perform(put("/api/bikes/")
```

Comment:
Why would you do a `put` on a URL without a Bike ID? Don't you use put to update an existing bike and have the ID in the URL?